nextcord/channel.py

```diff
@@ -67,24 +71,34 @@
 )
 
 if TYPE_CHECKING:
+    from . import (
+        embeds,
```

Comment:
Unused import `embeds` ?

Reply:
resolved in `eec0812`, also removed message and stickers